packages/legend-art/src/components/panel/Panel.tsx

```diff
@@ -0,0 +1,150 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { capitalize, prettyCONSTName, toTitleCase } from '@finos/legend-shared';
+import { clsx } from 'clsx';
+import { observer } from 'mobx-react-lite';
+
+export const PanelHeaderActionItem: React.FC<{
+  tip: string;
+  className?: string;
+  disabled?: boolean;
+  children: React.ReactNode;
+  onClick?: () => void;
+}> = (props) => {
+  const { className, onClick, children, disabled, tip } = props;
+  return (
+    <div className="panel__header__actions">
+      <button
+        className={clsx('panel__header__action', className)}
+        disabled={Boolean(disabled)}
+        onClick={onClick}
+        title={tip}
+      >
+        {children}
+      </button>
+    </div>
+  );
+};
+
+export const PanelHeader: React.FC<{
+  title: string;
+  children?: React.ReactNode;
+  className?: string;
+}> = (props) => {
+  const { title, children } = props;
+  return (
+    <div className="panel__header">
+      <div className="panel__header__title">
+        <div className="panel__header__title__label">{title.toLowerCase()}</div>
+      </div>
+      {children}
+    </div>
+  );
+};
+
+export const PanelTabs: React.FC<{
+  tabTitles: string[];
+  changeTheTab: <T>(
+    tab: T,
+  ) => (event: React.MouseEvent<HTMLDivElement>) => void;
+  selectedTab: string;
+  className?: string;
+  tabClassName: string;
+}> = (props) => {
+  const { tabTitles, changeTheTab, selectedTab, tabClassName } = props;
+
+  return (
+    <div className="panel__header">
+      <div className="panel__header__tabs">
+        {tabTitles.map((tab) => (
+          <div
+            key={tab}
+            onClick={changeTheTab(tab)}
+            className={clsx(tabClassName, {
+              [`${tabClassName}--active`]: tab === selectedTab,
+            })}
+          >
+            {toTitleCase(prettyCONSTName(tab))}
+          </div>
+        ))}
+      </div>
+    </div>
+  );
+};
+
+export const PanelForm: React.FC<{
+  children: React.ReactNode;
+  className?: string;
+}> = (props) => {
+  const { children } = props;
+  return <div className="panel__content__form">{children}</div>;
+};
+
+export const PanelContent: React.FC<{
+  children?: React.ReactNode;
+}> = (props) => {
+  const { children } = props;
+  return <div className="panel__content">{children}</div>;
+};
+
+export const Panel: React.FC<{
+  children?: React.ReactNode;
+}> = (props) => {
+  const { children } = props;
+  return <div className="panel__content__resizable">{children}</div>;
```

Comment:
I would expect this to be `className="panel"` only

Reply:
There is an existing panel though  `display: flex;
  flex-direction: column;` seems to work fine rn as well so I will delete the panel__resizable for now and leave it as panel